builder/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG BUILD_FROM
+FROM $BUILD_FROM
+
+ARG BUILD_ARCH
+
+# setup locals
+RUN apt-get update && apt-get install -y \
+        jq \
+    && rm -rf /var/lib/apt/lists/* \
+ENV LANG C.UTF-8
+
+# Install docker
+# https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
+RUN apt-get update && apt-get install -y \
```

Comment:
You probably can use `apt-get install -y --no-install-recommends`. 
Depends on the usage, but I got away with it. Makes the resulting image smaller.